src/plugins/credential_management/server/routes/create.ts

```diff
@@ -14,43 +14,43 @@ import { schema } from '@osd/config-schema';
 import { createHandler } from '../credential';
 
 export function registerCreateRoute(router: IRouter) {
```

Comment:
shall we just use saved objects APIs?

Reply:
> shall we just use saved objects APIs?

Plated it on a separate issue, as discussed offline.

https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1850